tests.py

```diff
@@ -0,0 +1,6 @@
+from test_connections import connection
```

Comment:
I don't really understand the reason for having this file

Reply:
I want to have a different file for each test format (one for the connection, one for functionalities an so on...) which will be activated by one "tests.py" file that will have the assertions.

this way the tests.py file will be clear and understood and the tests done will be well seen and if need be to remove a test will only require a line to be deleted as opposed to searching for a function in a messy file full of tests.